src/routes/workshop-a11y-fundamentals.svelte

```diff
@@ -203,174 +550,348 @@
 </svelte:head>
 
 <header class="header">
-  <ToggleTheme klass="toggleTheme" />
-  <!-- <Nav on:calculated={() => true} horizonSpace="10000" /> -->
+  <a href={SITE_URL} class="logo u-link">
+    Sandrina
+  </a>
+  <ToggleThemePage />
 </header>
 
-<main class="wrapper">
-  <h1 class="f-title title">Web Accessibility Fundamentals</h1>
-  <p class="rw">
-    Remote Workshop
-    <span class="rw-d">•</span>
-    <span class="rw-t">3 x 3hours</span>
+
+<main class="wrapper blogpost">
+  <header class="t-hero">
+    <h1 class="t-heroTitle t-title">Web Accessibility Fundamentals</h1>
+    <p class="t-heroMantra">The web is awesome and everyone should be able to enjoy it.</p>
+    <span class="t-heroKicker">Online Workshop</span>
+    <div class="t-heroAbout">
+      <!-- <p>April 11-13 · {eventHour} {eventTZ}</p> -->
+      <p class="t-heroCreator">with <img src="/sandrinap.jpg" alt=""  class="t-heroCreatorPic" />
+        <a class="u-link t-heroCreatorName" rel="noreferrer" href={TWITTER_URL} on:click={() => trackClick('csstricks_journey')}>
+          Sandrina Pereira
+        </a>
+      </p>
+    </div>
+  </header>
+
+  <div class="cta-sticky">
+    <!-- <button class="u-link">Coming soon!</button> -->
+  </div>
+
+  <span class="t-separator"></span>
+
+  <p class="t-p">
+    Let’s be honest, as web creators, we are still leaving accessibility as an afterthought. Learning how to create web apps using modern UI libraries sounds way more exciting, <i>I understand you...</i>
+  </p>
+
+  <h2 class="t-hook">
+      A11Y doesn't need to be boring
+  </h2>
+  
+    
+  <p class="t-p">
+    Perhaphs you already know that accessibility matters  and you are aware that color contrast is a thing.
```

Comment:
Perhaps you already know that accessibility matters and you are aware that color contrast is a thing.